lib/scheduler.js

```diff
@@ -118,9 +118,8 @@ class Scheduler extends EventEmitter {
     const masterKey = this.masterKey()
     if (!this.connection || !this.connection.redis) { return }
 
-    let lockedByMe = await this.connection.redis.setnx(masterKey, this.options.name)
+    let lockedByMe = await this.connection.redis.set(masterKey, this.options.name, 'NX', 'EX', this.options.masterLockTimeout)
     if (lockedByMe === true || lockedByMe === 1) {
```

Comment:
I think this test is no longer needed... might as well just always return true then.

Reply:
Other than this, the tests all pass!  Thank you for the update!